versions.props

```diff
@@ -48,3 +48,4 @@ com.esotericsoftware:kryo = 4.0.2
 org.eclipse.jetty:* = 9.4.43.v20210629
 org.testcontainers:* = 1.17.5
 io.delta:delta-core_* = 2.2.0
+com.github.luben:zstd-jni = 1.5.2-3
```

Comment:
looks like there's a newer version available ([v1.5.4-1](https://github.com/luben/zstd-jni/releases/tag/v1.5.4-1)), should we switch to that?

Reply:
Sure.